enclave-runtime/src/attestation.rs

```diff
@@ -92,11 +96,29 @@ pub fn create_ra_report_and_signature(
 		},
 	};
 
-	match attestation_handler.create_ra_report_and_signature(sign_type, skip_ra) {
-		Ok(r) => Ok(r),
-		Err(e) => {
-			error!("create_ra_report_and_signature failure: {:?}", e);
-			Err(e.into())
+	match remote_attestation_type {
+		RemoteAttestationType::Epid => {
+			match attestation_handler.create_epid_ra_report_and_signature(sign_type, skip_ra) {
+				Ok(epid) => Ok(epid),
+				Err(e) => {
+					error!("create_epid_ra_report_and_signature failure: {:?}", e);
+					Err(e.into())
+				},
+			}
+		},
+		RemoteAttestationType::Dcap => {
+			// FIXME: this shoudl return a result with value
```

Comment:
Can you explain that fixme? I don't understand it. :)

Reply:
That was for myself, when I was working on this branch, it became obsolete :).